x-pack/plugins/observability_solution/observability_onboarding/e2e/playwright/stateful/pom/components/header_bar.component.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { expect, Page } from '@playwright/test';
+
+export class HeaderBar {
+  page: Page;
+
+  constructor(page: Page) {
+    this.page = page;
+  }
+
+  public readonly helpMenuButton = () =>
+    this.page.locator('xpath=//div[@data-test-subj="helpMenuButton"]');
```

Comment:
Basically the same question as before, why use xpath?  I could be wrong but I recall it being rather slow in the past and we've got a ton of issues with resources in testing.